sample-app.yaml

```diff
@@ -0,0 +1,10 @@
+name: sample-app
+replicas: 1
+public: true
+containers:
+  app:
+    image: okteto/sample-app
+    ports:
+      - https:443:http:80
```

Comment:
This is fine, it is to work on the free tier

Reply:
It doesn't work on defaut clusters unless it has ingress. for now I'm going to go back to only http so it works for both